packages/app-store/routing-forms/pages/routing-link/[...appPages].tsx

```diff
@@ -213,8 +209,6 @@ export default function RoutingLink(props: inferSSRProps<typeof getServerSidePro
   return <RoutingForm {...props} />;
 }
 
-RoutingLink.isBookingPage = true;
```

Comment:
This can be removed because RoutingLink is the only component with `isBookingPage = true`, and I handled this by adding `/apps/routing-forms/routing-link` to `useIsBookingPageHook`